gateway/src/apicast/policy/cors/apicast-policy.json

```diff
@@ -46,7 +46,7 @@
         }
       },
       "allow_origin": {
-        "description": "Origin allowed for CORS requests. The field expects only one origin (e.g. https://example.com) or '*'. If left blank, the value of the 'Origin' request header will be used. In case of valid regexp, and if it matches with Origin header value, the value will be set to the Origin Value. In case of does not match, the header will not set at all.",
+        "description": "Origin allowed for CORS requests. The field expects only one origin (e.g. <code>https://example.com</code>) or '*'. If left blank, the value of the 'Origin' request header will be used. <br> In order to allow more than one origin it is possible to use a regular expression, if it matches with Origin header value, the value will be set to the Origin Value. In case it does not match, the header will not set at all. Example: <code>(api|web).test.com</code> wil match both <code>api.test.com</code> and <code>web.test.com</code>",
```

Comment:
It seems like GH picked the first suggestion instead of the second (corrected without html markup) :(

Reply:
Fixed now! Thanks!